doc/FUZZING.md

```diff
@@ -0,0 +1,59 @@
+# Fuzz testing
+
+C-lightning currently supports coverage-guided fuzz testing using [LLVM's libfuzzer](https://www.llvm.org/docs/LibFuzzer.html)
+when built with clang.
+
+
+## Build the fuzz targets
+
+In order to build the C-lightning binaries with code coverage you will need a recent
+[clang](http://clang.llvm.org/). The younger the compiler version the better.
+
+Then you'll need to enable support at configuration time. You likely want to enable
+a few sanitizers for bug detections as well as experimental features for an extended
+coverage (no required though).
+
+```
+DEVELOPER=1 EXPERIMENTAL_FEATURES=1 ASAN=1 UBSAN=1 VALGRIND=0 FUZZING=1 CC=clang ./configure && make
+```
+
+The targets will be built in `tests/fuzz/` as `fuzz-` binaries.
+
+
+## Run one or more target(s)
+
+You can run each target independently. Pass `-help=1` to see available options, for
```

Comment:
is this a library flag? if not you should use two dashes for flags! (we are not bitcoind). also it'd be nicer if you can make this not require a value.
```suggestion
You can run each target independently. Pass `--help` to see available options, for
```

Reply:
Library flag :/

> we are not bitcoind

x)

Reply:
Actually `--help` works like a charm :-)